client/my-sites/stats/stats-chart-tabs/utility.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import { rangeOfPeriod } from 'state/stats/lists/utils';
+
+export function formatDate( date, period ) {
+	const momentizedDate = moment( date );
+	switch ( period ) {
+		case 'day':
+			return momentizedDate.format( 'LL' );
+		case 'week':
+			return momentizedDate.format( 'L' ) + ' - ' + momentizedDate.add( 6, 'days' ).format( 'L' );
```

Comment:
this probably needs to be translated. I guess it was already like this but I think we shouldn't assume that `A - B` is appropriate

feel free to defer this for now but maybe earmark it for a small followup?